src/french.js

```diff
@@ -1,5 +1,5 @@
 const allInfo = {
-  tense: {
+  Tense: {
```

Comment:
could this be broken down? `tense: ['present', 'passe', ...]`, `type: ['indicatif', 'subjonctif'....]`